aea/test_tools/docker_image.py

```diff
@@ -113,7 +108,6 @@ def wait(self, max_attempts: int = 15, sleep_rate: float = 1.0) -> bool:
         :param sleep_rate: the amount of time to sleep between different requests.
         :return: True if the wait was successful, False otherwise.
         """
-        return True
```

Comment:
can never be covered by pytest

Reply:
Just add a no cover tag, don't update the code

Reply:
this is an abstractmethod

Reply:
Yes an abstract method doesn't need an implementation. So you can remove line 121